bitnami/memcached/templates/statefulset.yaml

```diff
@@ -0,0 +1,119 @@
+{{- if eq .Values.architecture "high-availability" }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "memcached.fullname" . }}
+  labels: {{- include "memcached.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels: {{- include "memcached.matchLabels" . | nindent 6 }}
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ template "memcached.fullname" . }}-headless
```

Comment:
Am I missing the manifest for this "headless" service?

Reply:
helm 3 failed without it. Not sure what it is about

Reply:
Added headless svc when doing ha / statefulset